packages/@misk/cli/src/index.ts

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env node
 
-import * as yargs from "yargs"
+import yargs = require('yargs');
```

Comment:
1) Could you run linting please? (should fix no semi-colons and double quote marks only) `$npm run-script lint`
2) In my tests, it looks like changing the import to `import yargs from "yargs"` is sufficient. Could you do that instead?
3) Could you also update the import statement in `src/utils/handleCommand.ts`?

Thanks for contributing!